forte/data/data_pack.py

```diff
@@ -785,17 +787,30 @@ def get_data(
                             "component": ["dummy"],
```

Comment:
btw, we may want to remove the "component" part form the example, people may not understand what this means.